lib/docker/docker_image.js

```diff
@@ -86,13 +86,17 @@ export default class DockerImage {
       pullOptions.authconfig = this.credentials;
     }
 
-    return await this.runtime.stats.timeGen(
-        'dockerImageDownloadTime',
-        this.pullImageStreamTo(this.runtime.docker,
-                               dockerImageName,
-                               this.stream,
-                               pullOptions)
+    let start = Date.now();
```

Comment:
I should add a generic `monitor.timethisfunction()` to `tc-lib-monitor`. https://github.com/taskcluster/taskcluster-lib-monitor#timing-handlers is similar but not quite what you want here I guess.